HPXMLtoOpenStudio/resources/constructions.rb

```diff
@@ -681,6 +835,10 @@ def self.apply_wood_frame_floor_ceiling(model, surfaces, constr_name, is_ceiling
           addtl_thick_in = rigid_r / 3.0 # Assume roughly R-3 per inch of loose-fill above cavity
           mat_addtl_ins = Material.new(name: 'ceiling loosefill ins', thick_in: addtl_thick_in, mat_base: BaseMaterial.InsulationGenericLoosefill, k_in: addtl_thick_in / rigid_r)
         end
+        mat_rb = nil
+        if has_radiant_barrier
+          mat_rb = Material.RadiantBarrier(radiant_barrier_grade, true)
+        end
```

Comment:
Why is this code inside the `else`? That means it will only apply if the ceiling has insulation. It should be applicable even if the ceiling is uninsulated. Comment applies to all ceiling methods.

Reply:
Now moved.